wolvic/wolvic_content_browser_client.cc

```diff
@@ -77,27 +65,11 @@ XrIntegrationClient* WolvicContentBrowserClient::GetXrIntegrationClient() {
 #endif
 
 std::string WolvicContentBrowserClient::GetUserAgent() {
-  typedef wolvic::SessionSettings::UserAgentMode UserAgentMode;
-
-  auto user_agent_override =
-      wolvic::SessionSettings::Get()->GetUserAgentOverride();
-  if (user_agent_override)
+  auto* settings = wolvic::SessionSettings::Get();
```

Comment:
Maybe a CHECK for settings having a value here could be good. Or just in wolvic::SessionSettings::Get().

Performance penalty should not be bad (we are not accessing GetUserAgent in a massively critical performance path), and it would crash immediately if we are doing something wrong with process startup/shutdown.

Reply:
There is already a DCHECK() in `Get()` precisely for that